.github/workflows/pending-review-notification.yml

```diff
@@ -22,9 +22,10 @@ jobs:
         with:
           python-version: '3.8.15'
           architecture: 'x64'
-      - uses: DubeySandeep/pending-review-notification@v1
+      - uses: oppia/stale-review-request-notifier@develop
```

Comment:
Can you use a commit hash here instead? It's a minor thing, but a commit hash is slightly safer in case somehow a malicious change makes it into the oppia/stale-review-request-notifier repository (which we probably won't scrutinize as closely as oppia/oppia)

For an example of how to do this, see 

https://github.com/oppia/oppia/blob/e9a37d317ded35f23781d6a26086fd0774a5b570/.github/workflows/revert-web-wiki-updates.yml#L12

Reply:
Thanks for pointing this out! done.